logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -106,6 +106,30 @@ public Queue(String dirPath, int pageCapacity, long maxBytes, CheckpointIO check
         }
     }
 
+    public String getDirPath() {
+        return dirPath;
```

Comment:
I personally use `this.`  on instance variables to emphasize that fact, I understand this might not be a univeral best-practice but I'd prefer if we could keep that uniform per class at least. Either we agree on removing all usage of `this.` which is not strictly necessary or we stay uniform within classes.